src/runtime/pkg/katautils/config.go

```diff
@@ -490,6 +494,34 @@ func (h hypervisor) getTxRateLimiterCfg() uint64 {
 	return h.TxRateLimiterMaxRate
 }
 
+func (h hypervisor) getNetRateLimiterBwMaxRate() uint64 {
+	return h.NetRateLimiterBwMaxRate
+}
+
+func (h hypervisor) getNetRateLimiterBwOneTimeBurst() uint64 {
+	if h.NetRateLimiterBwOneTimeBurst != 0 && h.getNetRateLimiterBwMaxRate() == 0 {
+		kataUtilsLogger.Warn("The NetRateLimiterBwOneTimeBurst is set but NetRateLimiterBwMaxRate is not set, this option will be ignored.")
+
+		h.NetRateLimiterBwOneTimeBurst = 0
+	}
```

Comment:
I'd consider it cleaner if this check (and other similar in this PR) were performed in a one-off sanity checking/consistency fixing section after a config is loaded instead of on each get.

OTOH I suspect that these gets aren't called very often so it might not make a huge difference in practice.  I still thought that I'd mention it anyway for sake of conceptual cleanliness.

Reply:
I've left this as it's as we have a few other getters doing the same thing already.